x-pack/plugins/data_quality/public/application.tsx

```diff
@@ -55,6 +55,7 @@ const AppWithExecutionContext = ({
         <PerformanceContextProvider>
           <Routes>
             <Route path="/" exact={true} render={() => <DatasetQualityRoute />} />
+            <Route path="/dataset" exact={true} render={() => <DatasetQualityDetailsRoute />} />
```

Comment:
[nit] I'd use `details` instead of `dataset`
```suggestion
            <Route path="/details" exact={true} render={() => <DatasetQualityDetailsRoute />} />
```